client/my-sites/stats/stats-reditect-flow/index.tsx

```diff
@@ -26,9 +36,26 @@ const StatsRedirectFlow = () => {
 		siteCreatedTimeStamp &&
 		new Date( siteCreatedTimeStamp ) > new Date( '2024-01-01' );
 
+	const isRequesting = useSelector( ( state: object ) => isStatsNoticeSettingsFetching( state ) );
+	const dispatch = useDispatch();
+
+	// Only runs on mount.
```

Comment:
This comment should probably be removed. My understanding is it's not technically correct now that the `useEffect` block has a dependancy array.